README.md

```diff
@@ -1,46 +1,173 @@
+*In this paper, we investigate the task of learning word embeddings from very sparse data in an incremental, cognitively-plausible way. We focus on the notion of informativeness, that is, the idea that some content is more valuable to the learning process than other. We further highlight the challenges of online learning and argue that previous systems fall short of implementing incrementality. Concretely, we incorporate informativeness in a previously proposed model of nonce learning, using it for context selection and learning rate modulation. We test our system on the task of learning new words from definitions, as well as on the task of learning new words from potentially uninformative contexts. We demonstrate that infor- mativeness is crucial to obtaining state-of-the-art performance in a truly incremental setup.*
 
 ## A note on the code
-We have had queries about *where* exactly the Nonce2Vec code resides. Since it is a modification of the original gensim Word2Vec model, it is located in the gensim/models directory, confusingly still under the name *word2vec.py*. All modifications described in the paper are implemented in that file. Note that there is no C implementation of Nonce2Vec, so the program runs on standard numpy. Also, only skipgram is implemented -- the cbow functions in the code are original Word2Vec.
+We have significantly refactored the original Nonce2Vec code in order to make replication easier and to make it work with gensim v3.x. You can use Nonce2Vec v2.x to replicate the results of the SRW paper. However, to replicate results of the original ENMLP paper, refer to Nonce2Vec v1.x found under the [emnlp2017 branch](https://github.com/minimalparts/nonce2vec/tree/release/emnlp2017) as we **cannot** guarantee fair replication between v1.x and v2.x.
 
+## Install
+You can install Nonce2Vec via:
+```bash
+pip3 install nonce2vec
+```
 
 ## Pre-requisites
-You will need a pre-trained gensim model. You can go and train one yourself, using the gensim repo at [https://github.com/rare-technologies/gensim](https://github.com/rare-technologies/gensim), or simply download ours, pre-trained on Wikipedia: 
-
-`wget http://clic.cimec.unitn.it/~aurelie.herbelot/wiki_all.model.tar.gz`
-
-If you use our tar file, the content should be unpacked into the models/ directory of the repo.
+To run Nonce2Vec, you need two gensim Word2Vec models (a skipgram model and a cbow model to compute informativeness-metrics). You can download the skipgram model from:
+```bash
+wget http://129.194.21.122/~kabbach/gensim.w2v.skipgram.model.7z
```

Comment:
Where do you unpack?

Reply:
Wherever you want: It's not longer specified since you can pass the absolute path to the model as argument of N2V via n2v --model /abs/path/to/model